ClothesApp/WebAPI/Authentication/ClaimExtractor.cs

```diff
@@ -0,0 +1,23 @@
+using System.Security.Claims;
+using Domain.Enums;
+using Infrastructure.Authentication;
+
+namespace WebAPI.Authentication;
+
+public static class ClaimExtractor
```

Comment:
claim extractor with ExtractedClaims returned model sounds like not a great english
first of all I would just make it extension method instead of static method
and also I would just split it into two methods GetUserId() and GetUserRole() instead of all these new models, etc..